### PR TITLE
feat!: YaruInfoBox visual improvements

### PR DESCRIPTION
**Pull request by ashuntu**
Adjusts the look of YaruInfoTiles. Notably it makes the text sizes all `bodyMedium`, aligns the icon to the top left, and adjusts some of the padding.

It also removes the inner `ListTile` since it was causing more issues than its worth to achieve our style goals. This is a breaking change since it removes the `isThreeLine` field, but that field isn't really relevant without the underlying `ListTile`. This could potentially be included in https://github.com/ubuntu/yaru.dart/issues/1057 when a new tile is added, but for now I think this stands on its own.

Also adds some tests for this widget as there were none before.

|       | Before | After |
|-------|--------|-------|
| Light | image | image |
| Dark  | image | image |

cc @juanruitina 

---

UDENG-8485

### CODE REVIEW

**Comment by Feichtmeier** [comment]
That looks indeed better! Thank you

If you changed the fields of the class I consider this a breaking change and you need to insert the exclamation ❗ after feat in the title/commit so the release bot bumps a major version

File: lib/src/widgets/yaru_info.dart [hunk 1]
```diff
@@ -104,28 +103,21 @@ class YaruInfoBox extends StatelessWidget {
...
 
-  /// An optional trailing [Widget] inserted into the internal [ListTile].
+  /// An optional trailing [Widget] inserted to the right of the title.
   final Widget? trailing;
 
-  ///  Whether the internal [ListTile] is intended to display three lines of text.
-  /// If true, then [subtitle] must be non-null (since it is expected to give the second and third lines of text).
-  /// If false, the list tile is treated as having one line if the subtitle is null and treated as having two lines if the subtitle is non-null.
-  /// When using a [Text] widget for [title] and [subtitle], you can enforce line limits using [Text.maxLines].
-  final bool isThreeLine;
```
Comment (Feichtmeier) [inline lib/src/widgets/yaru_info.dart:127]: Small but Breaking change :)
